portal/models/fhir.py

```diff
@@ -572,6 +572,30 @@ def aggregate_responses(instrument_ids, current_user):
 
     return bundle
 
+
+def qnr_document_id(
+        subject_id, questionniare_bank_id, questionnaire_name, status):
+    """Return document['identifier'] for matching QuestionnaireResponse
+
+    Using the given filter data to look for a matching QuestionnaireResponse.
+    Expecting to find exactly one, or raises NoResultFound
+
+    :return: the document identifier
```

Comment:
The identifier is a string, could you add `:rtype: str`?

Reply:
This is python, we don't type things unless strictly necessary.  The code will work for either type, whatever is stored in the json.

Reply:
Fair enough, this is just more boilerplate to keep up to date when code changes. 
Any strong feelings about type-hinting? :)